hexgame/game.py

```diff
@@ -28,8 +28,7 @@ def __init__(
 
     def _play(self) -> None:
         # player  move
-        new_board = self.current_player.play(self.board)
-        self.board = new_board
+        self.current_player.play(self.board)
```

Comment:
Now that we have done it in this way, I start to like less this player.play(board). 

You can say the issue was already there but it is making it more obvious for me. What I do not like: in reality all this code could have been done in "a pure functional" way (a board goes in and a board goes out) which might be my justification for why I was okay with that approach.

 Now we are clearly not doing that anymore and we are embracing OOP, as we should.

But: I do not like now that player, has a function play that has a side effect on another object, the board. This is not wrong and is more of philosophy thing, but I just don't like this muddling of concerns.

So my proposal: let's add a player.get_move fucnction that spit a move, then in board._play we will pass the move and call place_stone. This way the only side effects for the board are managed by methdos of the board.

Sorry for long answer but I thought this was an important matter.